alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/TestHelpers.kt

```diff
@@ -111,6 +114,39 @@ fun randomTraditionalTrigger(
         actions = if (actions.isEmpty()) (0..randomInt(10)).map { randomAction(destinationId = destinationId) } else actions)
 }
 
+fun randomAggregationTrigger(
+    id: String = UUIDs.base64UUID(),
+    name: String = ESRestTestCase.randomAlphaOfLength(10),
+    severity: String = "1",
+    bucketSelector: BucketSelectorExtAggregationBuilder = randomBucketSelectorExtAggregationBuilder(name=name),
```

Comment:
NP: To be consistent, can you add spaces around the `=` in the named argument (same for script in `randomBucketSelectorExtAggregationBuilder` below).